### PR TITLE
Add QR Scanner block.

### PR DESCRIPTION
**Pull request by StephanieJKS**
Closes ~#ISSUE_NUMBER~

### What are the changes and their implications?
This PR adds a QR scanner block to the default Lowdefy blocks.

## Checklist

- [x] Pull request is made to the "develop" branch
- [ ] Tests added
- [ ] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

File: packages/plugins/blocks/blocks-qr/src/blocks/QRScanner/QRScanner.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    );
+  }
+
+  onStop() {
+    this.html5QrCode?.stop();
+  }
+
+  componentDidMount() {
+    this.html5QrCode = new Html5Qrcode(this.props.blockId);
+    if (!this.props.properties.inactiveByDefault) {
```
Comment (Gervwyk) [inline packages/plugins/blocks/blocks-qr/src/blocks/QRScanner/QRScanner.js:48]: Please add this property to readme.

**Review by Gervwyk — COMMENTED** [review]
Nice! Thanks for the readme!

File: packages/plugins/blocks/blocks-qr/src/blocks/QRScanner/QRScanner.js (hunk 1 above)
Comment (StephanieJKS) [reply packages/plugins/blocks/blocks-qr/src/blocks/QRScanner/QRScanner.js:48]: Change made in cb4b780187346c167ced524eea1d1d149549ee79

File: packages/plugins/blocks/blocks-qr/src/blocks/QRScanner/schema.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        "default": false,
+        "description": "Determines whether or not the scanner is started when it is mounted."
+      },
+      "formatsToSupport": {
+        "type": "array",
+        "description": "List of Html5QrcodeSupportedFormats to support. See <a href='https://github.com/mebjas/html5-qrcode#documentation:~:text=library.%0A%20*/%0Aenum-,Html5QrcodeSupportedFormats,-%7B%0A%20%20QR_CODE%20%3D'>more</a>.",
+        "items": {
+          "type": "number",
+          "enum": [
+            "QR_CODE = 0",
```
[the remark this answers is not in the document]
Comment (StephanieJKS) [reply packages/plugins/blocks/blocks-qr/src/blocks/QRScanner/schema.json:35]: Wasn't sure about this on to be honest. @Gervwyk what do you think?

**Review by SamTolmay — APPROVED** [review]

**Pull request merged**